Dockerfile

```diff
@@ -9,14 +10,25 @@ COPY . .
 
 RUN ./autogen.sh && \
     if [ "$TARGETARCH" = "arm64" ]; then BUILD_HOST="aarch64-linux-gnu"; else BUILD_HOST="x86_64-pc-linux-gnu"; fi && \
-    ./configure --prefix=/tapyrus-core/depends/$BUILD_HOST --enable-cxx --disable-shared --disable-replication --with-pic --with-incompatible-bdb $TAPYRUS_CONFIG && \
+    ./configure --prefix=/tapyrus-core/depends/$BUILD_HOST  $TAPYRUS_CONFIG && \
     make -j"$(($(nproc)+1))" && \
     make install
 
 FROM ubuntu:20.04
 
 COPY --from=builder /tapyrus-core/dist/bin/* /usr/local/bin/
 
+ENV TRACE_PACKAGES "ssh bpfcc-tools bpftrace python3-bpfcc"
+
+RUN apt-get update
+RUN apt-get install --no-install-recommends --fix-missing --no-upgrade -qq $TRACE_PACKAGES
+
+WORKDIR /kernel
+COPY  --from=ksrc /kernel-dev.tar .
+RUN tar xf kernel-dev.tar
+RUN mv /kernel/usr/src/linux-headers* /kernel/usr/src/linux-headers
+ENV BPFTRACE_KERNEL_SOURCE=/kernel/usr/src/linux-headers
+
```

Comment:
I appreciate your work, but but from this PR, let's not apply USDT to container.